packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import UsingKeyboardProvider from './UsingKeyboardProvider';
+import BaseFontSizeProvider from './BaseFontSizeProvider';
```

Comment:
Thought: export `BaseFontSizeProviderProps` as well and then have `type LeafyGreenProviderProps = { children: React.ReactNode } & BaseFontSizeProviderProps;`

Reply:
👍